<ui version="4.0" >
 <class>dlgMain</class>
 <widget class="QWidget" name="dlgMain" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="palette" >
   <palette>
    <active>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>119</red>
        <green>119</green>
        <blue>119</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>159</red>
        <green>159</green>
        <blue>159</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>119</red>
        <green>119</green>
        <blue>119</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>159</red>
        <green>159</green>
        <blue>159</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>119</red>
        <green>119</green>
        <blue>119</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>247</red>
        <green>247</green>
        <blue>247</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>119</red>
        <green>119</green>
        <blue>119</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>159</red>
        <green>159</green>
        <blue>159</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>119</red>
        <green>119</green>
        <blue>119</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>119</red>
        <green>119</green>
        <blue>119</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>239</red>
        <green>239</green>
        <blue>239</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText" >
      <brush brushstyle="SolidPattern" >
       <color alpha="255" >
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle" >
   <string>KBFX Configurator Application</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/appicons/images/hi128-app-kbfx.png</normaloff>:/appicons/images/hi128-app-kbfx.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QListWidget" name="listWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="MinimumExpanding" hsizetype="Minimum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>265</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize" >
        <size>
         <width>150</width>
         <height>330</height>
        </size>
       </property>
       <property name="palette" >
        <palette>
         <active>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>239</red>
             <green>239</green>
             <blue>239</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>239</red>
             <green>239</green>
             <blue>239</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Base" >
           <brush brushstyle="SolidPattern" >
            <color alpha="255" >
             <red>239</red>
             <green>239</green>
             <blue>239</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="showDropIndicator" stdset="0" >
        <bool>false</bool>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="iconSize" >
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="textElideMode" >
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="viewMode" >
        <enum>QListView::ListMode</enum>
       </property>
       <property name="selectionRectVisible" >
        <bool>false</bool>
       </property>
       <item>
        <property name="text" >
         <string>Themes</string>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >
          <normaloff>:/categories/images/display.png</normaloff>:/categories/images/display.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Kicker Button</string>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >
          <normaloff>:/categories/images/kcharselect.png</normaloff>:/categories/images/kcharselect.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Fonts</string>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >
          <normaloff>:/categories/images/fonts.png</normaloff>:/categories/images/fonts.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Layout</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>:/Themes/images/messagebox_info.png</normaloff>:/Themes/images/messagebox_info.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Plugins</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>:/Themes/images/messagebox_info.png</normaloff>:/Themes/images/messagebox_info.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Tooltip</string>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >
          <normaloff>:/categories/images/konversation.png</normaloff>:/categories/images/konversation.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Menu Style</string>
        </property>
        <property name="icon" >
         <iconset>
          <normaloff>:/Themes/images/messagebox_info.png</normaloff>:/Themes/images/messagebox_info.png</iconset>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>About</string>
        </property>
        <property name="icon" >
         <iconset resource="images.qrc" >
          <normaloff>:/categories/images/messagebox_info.png</normaloff>:/categories/images/messagebox_info.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Maximum" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>150</width>
         <height>200</height>
        </size>
       </property>
       <property name="text" >
        <string>&lt;html>
&lt;head>&lt;/head>
&lt;body>
&lt;center>
&lt;b>Latest KBFX news&lt;/b>
&lt;br/>&lt;br/>
KBFX Silk 0.4.9.3.1 is out.
&lt;br/>&lt;br/>
it is the STABLE KBFX release
&lt;br/>&lt;br/>
See details &lt;a href="http://www.google.com">-HERE-&lt;/a>
&lt;br/>&lt;br/>
Please report bugs on this version &lt;a href="http://www.google.com">-HERE-&lt;/a>
&lt;/center>
&lt;/body>
&lt;/html></string>
       </property>
       <property name="wordWrap" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>5</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="sizeConstraint" >
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="lbl_menu_top" >
         <property name="minimumSize" >
          <size>
           <width>500</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #90b8d7, stop: 1 #0c669d);
color: #ffffff;</string>
         </property>
         <property name="text" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt; font-weight:600;">KBFX Themes&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="indent" >
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="sizeConstraint" >
          <enum>QLayout::SetMinimumSize</enum>
         </property>
         <item>
          <widget class="QLabel" name="lbl_menu_content" >
           <property name="minimumSize" >
            <size>
             <width>425</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>16777215</width>
             <height>75</height>
            </size>
           </property>
           <property name="styleSheet" >
            <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 1 #e2e2cc);</string>
           </property>
           <property name="text" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;">&lt;b>Tip!&lt;/b> To keep it simple place all your themes inside one folder.&lt;br/>Then create different folders inside that folder with different names.&lt;br/>KBFX will then read those folders as themes.&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="margin" >
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_menu_picture" >
           <property name="minimumSize" >
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>75</width>
             <height>75</height>
            </size>
           </property>
           <property name="styleSheet" >
            <string notr="true" >border-radius: 0px;
border: none;
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #ffffff, stop: 1 #e2e2cc);</string>
           </property>
           <property name="text" >
            <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;img width="70" height="70" src=":/categories/images/digikam.png" />&lt;/p>&lt;/body>&lt;/html></string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="page" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>504</width>
          <height>356</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QPushButton" name="pushButton_3" >
       <property name="text" >
        <string>&amp;Help</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/buttons/images/help.png</normaloff>:/buttons/images/help.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="text" >
        <string>&amp;Default</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton" >
       <property name="text" >
        <string>&amp;Apply</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/buttons/images/button-ok.png</normaloff>:/buttons/images/button-ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4" >
       <property name="text" >
        <string>E&amp;xit</string>
       </property>
       <property name="icon" >
        <iconset resource="images.qrc" >
         <normaloff>:/buttons/images/button_cancel.png</normaloff>:/buttons/images/button_cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections/>
</ui>
